aten/src/ATen/CMakeLists.txt

```diff
@@ -148,7 +148,10 @@ endif(MKLDNN_FOUND)
 
 list(APPEND ATen_CPU_DEPENDENCY_LIBS cpuinfo)
 
-if(NOT MSVC AND NOT EMSCRIPTEN)
+hunter_add_package(sleef)
+find_package(sleef CONFIG REQUIRED)
+list(APPEND ATen_CPU_DEPENDENCY_LIBS sleef::sleef)
+if(FALSE AND NOT MSVC AND NOT EMSCRIPTEN)
```

Comment:
Maybe it's better with `HUNTER_ENABLED`?

```cmake
if(NOT MSVC AND NOT EMSCRIPTEN AND NOT HUNTER_ENABLED)
```

Reply:
Even without HUNTER_ENABLED, they won't have third_party submodules right?
If the use case is they do, then I've already broken everywhere else that uses the submodules.

Reply:
> broken everywhere else that uses the submodules

`if(HUNTER_ENABLED` just looks saner than `if(FALSE` for me. Up to you.